prowler/lib/check/models.py

```diff
@@ -54,6 +54,7 @@ class Check_Metadata_Model(BaseModel):
     DependsOn: list[str]
     RelatedTo: list[str]
     Notes: str
+    Tags: dict = {}
```

Comment:
Why this? Sorry for the noise, in the previous comment I thought that this was being modified in another object.

Reply:
True